Jenkinsfile

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env groovy
+
+pipeline {
+    agent {
+        label "java"
+    }
+    tools {
+        maven 'mvn'
+        jdk 'jdk8'
+    }
+    options {
+        timestamps()
+    }
+
+    stages {
+        stage('Checkout') {
+            steps {
+                deleteDir()
+                checkout scm
+            }
+        }
+        stage('Test') {
+            steps {
+                sh 'mvn clean test'
```

Comment:
At least add `-B`; also `-ntp` after [INFRA-2129](https://issues.jenkins-ci.org/browse/INFRA-2129).

Reply:
I'll leave the `-ntp` flag as a future change until that version of maven has not been updated in the infra side.